wp-content/mu-plugins/pantheon/pantheon-try-gutenberg-mods.php

```diff
@@ -0,0 +1,54 @@
+<?php
+function _pantheon_try_gutenberg_host_link(){
+?>
+<style type="text/css">
+    .try-gutenberg-panel .try-gutenberg-panel-column:not(.try-gutenberg-panel-image-column) {
+        grid-template-rows: auto;
+    }
+    .try-gutenberg-panel .host-link {
+        font-weight: bold;
+    }
+    .try-gutenberg-action > p:first-of-type {
+        display: none;
```

Comment:
This is causing issues removing the _Install Gutenberg_ button when it is shown rather than the desired link so I am going to remove it.